staging/src/k8s.io/client-go/util/jsonpath/parser.go

```diff
@@ -85,7 +85,7 @@ func (p *Parser) Parse(text string) error {
 	return p.parseText(p.Root)
 }
 
-// consumeText return the parsed text since last cosumeText
+// consumeText return the parsed text since last consumeText
```

Comment:
While we are here, can you change this to `// consumeText returns the parsed text since the last consumed text.`?

Reply:
@nikhita Thank you! I have corrected it.